test_vm/src/lib.rs

```diff
@@ -276,6 +294,10 @@ impl<'bs> VM<'bs> {
         self.store.get_cbor::<C>(&a.head).unwrap()
     }
 
+    pub fn get_epoch(&self) -> ChainEpoch {
```

Comment:
This is here too https://github.com/filecoin-project/builtin-actors/blob/master/test_vm/src/lib.rs#L370

Reply:
Yeah, didn't rebase on master recently. Good thing its a small function